packages/derived-wallet-ethereum/src/EIP1193DerivedSignature.ts

```diff
@@ -3,30 +3,37 @@ import { Deserializer, Hex, HexInput, Serializer, Signature } from '@aptos-labs/
 export class EIP1193DerivedSignature extends Signature {
   static readonly LENGTH = 65;
 
+  // The signature of the message
   private readonly _siweSignature: Uint8Array;
+  // The date and time when the signature was issued
   readonly issuedAt: Date;
+  // The scheme in the URI of the message, e.g. the scheme of the website that requested the signature (http:, https:, etc.)
+  readonly scheme: string;
```

Comment:
The scheme should not contain the semicolon. That's just something [the `URL` class does and calls "protocol"](https://url.spec.whatwg.org/#dom-url-protocol:~:text=The%20protocol%20getter%20steps%20are%20to%20return%20this%E2%80%99s%20URL%E2%80%99s%20scheme%2C%20followed%20by%20U%2B003A%20(%3A).)

Reply:
thoughts on making this optional?
And when `undefined`, it will assumed to be `https`.
This way we have full coverage of schemes, but the signature is as short as it can be since the main use case is https and will occur like 90%+ times